### PR TITLE
Add ForbidAsync overload for automatic authentication scheme.

### PR DESCRIPTION
**Pull request by pranavkm**
cc @HaoK

### CODE REVIEW

File: src/Microsoft.AspNet.Http.Abstractions/Authentication/AuthenticationManager.cs [hunk 1]
```diff
@@ -99,6 +99,14 @@ public virtual Task ForbidAsync(string authenticationScheme, AuthenticationPrope
             return ChallengeAsync(authenticationScheme, properties, ChallengeBehavior.Forbidden);
         }
 
+        /// <summary>
+        /// Creates a challenge for the authentication manager with a <see cref="ChallengeBehavior.Forbidden"/>.
+        /// </summary>
+        /// <param name="properties">Additional arbitrary values which may be used by particular authentication types.</param>
+        /// <returns>A <see cref="Task"/> that represents the asynchronous challenge operation.</returns>
+        public virtual Task ForbidAsync(AuthenticationProperties properties)
```
Comment (Tratcher) [inline src/Microsoft.AspNet.Http.Abstractions/Authentication/AuthenticationManager.cs]: Should there also be ForbidAsync with no params?

Comment (HaoK) [inline src/Microsoft.AspNet.Http.Abstractions/Authentication/AuthenticationManager.cs]: Yeah if there is one with just properties. The empty one is likely to be more useful.

Comment (pranavkm) [inline src/Microsoft.AspNet.Http.Abstractions/Authentication/AuthenticationManager.cs]: Updated

**Comment by HaoK** [comment]
:shipit:

**Pull request closed without merging**